questionnaire/form-helper.js

```diff
@@ -165,7 +167,13 @@ function correctPartialDates(body, questionId) {
 }
 
 function addPrefix(section) {
-    return sectionList[section];
+    if (section === 'system') {
+        return section;
+    }
+    if (section.substring(0, 4) === 'info') {
+        return section.replace('info', 'p-');
```

Comment:
This can communicate that it should only replace "info-" at the start of a string using a regex e.g.

`return section.replace(/^info-/, 'p--');`

Given the name of function, it should already be obvious, however there's no harm in communicating intent

Reply:
Changed replace to startsWith and added regexes where appropriate